.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,18 @@
 <!-- 
 Describe the changes you have made here: what, why, ... 
-Link issues by using the following pattern: #333.
-If you fixed a koppor issue, link it with following pattern: [koppor#47](https://github.com/koppor/jabref/issues/47).
```

Comment:
Since fixing koppor issues is seldom, you ask the core developers to apply the linking style after merging the pull requests?

Reason: I would like to have the changelog rendered as

    - When an entry is fetched using a DOI, the DOI appears once. Fixes [koppor#8]`

Instead of

    - When an entry is fetched using a DOI, the DOI appears once. Fixes https://github.com/koppor/jabref/issues/8`

Similar question arises from normal issues. With the proposed changed syntax, issues won't be autolinked anymore (with plain GitHub without any plugins)

![grafik](https://user-images.githubusercontent.com/1366654/78522968-7a2e1f00-77cf-11ea-97bf-48d184c7dd50.png)

Reply:
This information is about the body of the pull request not about the changelog, right?